benchmark/README.md

```diff
@@ -0,0 +1,84 @@
+# AnnData Benchmarks
+
+This repo contains some work in progress benchmarks for [AnnData](https://github.com/theislab/anndata) using [asv](https://asv.readthedocs.io).
+
+## Setup
+
+I definitley recommend reading through the asv docs. Currently, this assumes the benchmark suite can reach the `anndata` repo via the path `../anndata`. Otherwise, all you'll need to do is create a [machine file](https://asv.readthedocs.io/en/stable/commands.html#asv-machine) for your system and make sure `anndata`s dependencies are installable via `conda`.
+
+### Data
+
+Data will need to be retrieved for these benchmarks. Currently this is can be done via the script `fetch_datasets.py`. This downloads and caches some files for each dataset. The set of datasets retrieved can be limited via the `--pattern` argument.
```

Comment:
I think this part can be removed along with the datasets folders. Maybe the whole section?